dhall/dhall.cabal

```
@@ -688,6 +688,7 @@ Test-Suite tasty
         tasty-quickcheck          >= 0.9.2    && < 0.11,
         tasty-silver                             < 3.3 ,
         template-haskell                               ,
+        temporary                 >= 1.2.1    && < 1.3 ,
```

The `stack.yaml` and the Nix config seem to provide `temporary-1.3` (https://hackage.haskell.org/package/temporary-1.3/changelog).
```suggestion
        temporary                 >= 1.2.1    && < 1.4 ,
```